src-vue/src/views/PlayView.vue

```diff
@@ -0,0 +1,126 @@
+<script lang="ts">
+import { ElNotification } from 'element-plus';
+import {Tabs} from "../utils/Tabs";
+import PlayButton from '../components/PlayButton.vue';
+import { defineComponent } from "vue";
+
+export default defineComponent({
+    data() {
+        return {
+            developerModeClicks: 0
+        }
+    },
+    components: {
+        PlayButton
+    },
+    computed: {
+        northstarIsRunning(): boolean {
+            return this.$store.state.northstar_is_running;
+        },
+        northstarVersion(): string {
+            return this.$store.state.installed_northstar_version;
+        },
+    },
+    methods: {
+        activateDeveloperMode() {
+            this.developerModeClicks += 1;
+            if (this.developerModeClicks === 6) {
```

Comment:
It should be five clicks >:(

Jokes aside I'm paranoid so I always use `>==` for bounds check in case that value gets skipped for some inexplicable reason :P

Reply:
Fixed with c508c62e8c098388cbcae942182c4909810beb70